### DISCUSSION TITLE
AL-Go for on-prem BC?

### DISCUSSION THREAD

**Discussion by njackson1582** [microsoft/AL-Go#123] [Q&A] [answered]
How could we leverage AL-Go for on-prem deployments of BC? Obviously we wouldn't be able to auto-deploy directly into the clients' environment, but could we still auto-build the app? 

What modifications to the repo would be required to support on-prem BC deployments?

**Comment by freddydk** [comment, marked as answer]
The reason why we state that Onprem is not supported is, that it potentially opens up a huge can of worms.

If your Onprem app is nice, no code customizations, no DLLs etc. - then you can just use the PTE template and add a setting called **enablePerTenantExtensionCop** and set it to false.
You would also need to create a LicenseFileUrl secret (like with appsource apps).
Then I think you should be good to go.

**Reply by njackson1582** [reply]
Great, thank you!
